src/vivarium_cluster_tools/vadmin/cli.py

```diff
@@ -28,3 +28,36 @@ def parse(logs_directory, result_directory, verbose):
     if not result_directory:
         result_directory = logs_directory
     log_parser.parse_log_directory(logs_directory, result_directory)
+
+
+@vadmin.group()
+def oauth():
+    """Add or delete OAuth tokens from github and stash.
+
+    This command manages authentication for github and stash by communicating
+    with the webservices over http through their respective REST APIs. It
+    allows a user to create, store, or delete OAuth tokens which will be
+    used in place of a user name and password when managing repositories.
+
+    """
+    pass
+
+
+@oauth.command()
+@click.argument('service', type=click.Choice(['stash', 'github']))
+@click.option('-v', 'verbose', count=True, help='Configure logging verbosity.')
+def create(service, verbose):
+    """Create a new OAuth token for a SERVICE."""
+    utilities.configure_master_process_logging_to_terminal(verbose)
+    repositories.oauth_create(service)
+
+
+@oauth.command()
+@click.argument('service', type=click.Choice(['stash', 'github']))
+def remove(service):
+    print(f'creating oauth token for {service}')
```

Comment:
You're adding this later, right? the holdover message says create

Reply:
Yep